geode-core/src/test/java/org/apache/geode/test/dunit/examples/LocatorPortClusterExampleTest.java

```diff
@@ -65,9 +65,10 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void clusterHasSixVMsByDefault() throws Exception {
-    assertThat(cache.getDistributionManager().getViewMembers()).hasSize(6);
-    assertThat(Host.getHost(0).getVMCount()).isEqualTo(4);
-    assertThat(Host.getHost(0).getAllVMs()).hasSize(4);
+  public void clusterHasDUnitVMCountPlusTwoByDefault() throws Exception {
+    int dunitVMCount = Host.getHost(0).getVMCount();
+    assertThat(cache.getDistributionManager().getViewMembers()).hasSize(dunitVMCount + 2);
+    assertThat(Host.getHost(0).getVMCount()).isEqualTo(dunitVMCount);
```

Comment:
This assert doesn't seem useful since the expected `dunitVMCount` is initialized with the same expression that is being tested.